schemas/iso19139/src/main/plugin/iso19139/convert/functions.xsl

```diff
@@ -191,8 +191,21 @@
   <xsl:template name="defaultTitle">
     <xsl:param name="isoDocLangId"/>
 
-    <xsl:variable name="poundLangId"
-                  select="concat('#',upper-case(java:twoCharLangCode($isoDocLangId)))"/>
```

Comment:
I think the change will cause issues with this code in `update-fixed-info` that uses the 2 letter code updating the metadata when saved to use the 2 letter code:

https://github.com/geonetwork/core-geonetwork/blob/96e590a420229a9f43d75a7c85126c5f930990a8/schemas/iso19139/src/main/plugin/iso19139/update-fixed-info.xsl#L616-L617

Not sure if maybe requires another PR to be merged also?